adal/src/main/java/com/microsoft/aad/adal/Discovery.java

```diff
@@ -228,7 +229,10 @@ private void performInstanceDiscovery(final URL authorityUrl, final String trust
             final Map<String, String> discoveryResponse = sendRequest(queryUrl);
             AuthorityValidationMetadataCache.processInstanceDiscoveryMetadata(authorityUrl, discoveryResponse);
             if (!AuthorityValidationMetadataCache.containsAuthorityHost(authorityUrl)) {
-                AuthorityValidationMetadataCache.updateInstanceDiscoveryMap(authorityUrl.getHost(), new InstanceDiscoveryMetadata(true));
+                ArrayList<String> aliases = new ArrayList<String>();
```

Comment:
Should we do the same change in (com/microsoft/aad/adal/AcquireTokenRequest.java:226)?

Reply:
Never mind, I think we don't need it for invalidated authority.